Added usage of aws session token to use temporary credentials. (#250)
django-ses/django-ses: README.rst

  *Required.* Alternative API keys for Amazon SES. This is useful in situations
  where you would like to use separate access keys for different AWS services.

``AWS_SES_SESSION_TOKEN``, ``AWS_SES_SECRET_ACCESS_KEY``

>   Optional. Use one of this variables, to provide session token that is required
>   when temporary credentials are used. Details:

Reviewer: ```suggestion
  Optional. Use `AWS_SES_SESSION_TOKEN` to provide session token
  when temporary credentials are used. Details:
```
